reference/conanfile/tools/apple.rst

```diff
@@ -127,4 +131,148 @@ option:
+        settings = "os", "arch", "compiler", "build_type"
+
+        def generate(self):
+            tc = XcodeToolchain(self)
+            tc.generate()
+
+
+The ``XcodeToolchain`` will generate three files after a ``conan install`` command. As
+explained above for the XcodeDeps generator, each different configuration will create a
+set of files with different names. For example, running ``conan install`` for *Release*
+first and then *Debug* configuration:
+
+.. code-block:: bash
+
+    $ conan install conanfile.py # default is Release
+    $ conan install conanfile.py -s build_type=Debug
+
+Will create these files:
+
+.. code-block:: bash
+
+    .
+    ├── conan_config.xcconfig
+    ├── conantoolchain_release_x86_64.xcconfig
+    ├── conantoolchain_debug_x86_64.xcconfig
+    └── conantoolchain.xcconfig
+
+Those files are:
+
+- The main *conan_config.xcconfig* file, to be added to the project. Includes both the
+  files from this generator and the generated by the
+  :ref:`XcodeDeps<conan_tools_apple_xcodedeps>` in case it was also set.
+- *conantoolchain_<debug/release>_x86_64.xcconfig*: declares ``CLANG_CXX_LIBRARY``,
+  ``CLANG_CXX_LANGUAGE_STANDARD`` and ``MACOSX_DEPLOYMENT_TARGET`` variables with
+  conditional logic depending on the build configuration, architecture and sdk set.
+- *conantoolchain.xcconfig*: aggregates all the *conantoolchain_<config>_<arch>.xcconfig*
+  files for the different installed configurations.
+
+
+Every invocation to ``conan install`` with different configuration will create a new
+*conantoolchain_<config>_<arch>.xcconfig* file that is aggregated in the
+*conantoolchain.xcconfig*, so you can have different configurations included in your Xcode
+project.
+
+The XcodeToolchain files can declare the following Xcode build settings based on Conan settings values:
+
+- ``MACOSX_DEPLOYMENT_TARGET`` is based on the value of the ``os.version`` setting and
+  will make the build system to pass the flag ``-mmacosx-version-min`` with that value (if
+  set). It defines the operating system version the binary should run into.
+- ``CLANG_CXX_LANGUAGE_STANDARD`` is based on the value of the ``compiler.cppstd`` setting
+  that sets the C++ language standard.
+- ``CLANG_CXX_LIBRARY`` is based on the value of the compiler.libcxx setting and sets the
+  version of the C++ standard library to use.
+
+One of the advantages of using toolchains is that they can help to achieve the exact same build
+with local development flows, than when the package is created in the cache.
+
+
+XcodeBuild
+----------
+
+Available since: `1.46.0 <https://github.com/conan-io/conan/releases>`_
+
+The ``Xcode`` build helper is a wrapper around the command line invocation of Xcode. It
+will abstract the calls like ``xcodebuild -project app.xcodeproj -configuration <config>
+-arch <arch> ...``
+
+The ``Xcode`` helper can be used like:
+
+.. code:: python
+
+    from conan import conanfile
+    from conan.tools.apple import XcodeBuild
```

Comment:
This seems wrong, importing ``XcodeBuild``, using ``Xcode``

Reply:
But ``Xcode`` is not imported anywhere, what is imported is ``XcodeBuild``, this would fail

Reply:
Right, I thought you talked about the name 🤦 , I'll fix that